src/app/components/molecules/Input/Input.tsx

```diff
@@ -31,8 +32,10 @@ export interface InputProps {
     autoFocus?: boolean;
     children?: never;
     className?: string;
+    colorNegativeAmount?: string;
```

Comment:
why do we need it? can it be a different color other then the color in the theme?

Reply:
After discussion with the designer, decided to not have an optional color